client/test/preview/unit/jest.setup.ts

```diff
@@ -1,9 +0,0 @@
-import '@testing-library/jest-dom';
-import 'test/preview/__mocks__/global_mocks';
```

Comment:
I guess this file will be needed once the global mocks for preview code come into the `test/preview ` directory.